packages/shared/src/components/cards/ShowMoreContent.tsx

```diff
@@ -42,7 +42,7 @@ export default function ShowMoreContent({
         {getContent()}{' '}
         {displayShowMoreLink() && (
           <ClickableText
-            className="inline-flex text-theme-label-link"
+            className="inline-flex !text-theme-label-link"
```

Comment:
Do we need to do anything with hover and focus states? Since this now trumps all...

Reply:
Think they always keep being this
the logic was already there.